plasmapy/diagnostics/thomson.py

```diff
@@ -1000,10 +1000,12 @@ def spectral_density_model(  # noqa: C901, PLR0912, PLR0915
     #       quantities isn't consistent with the number of that species defined
     #       by ifract or efract.
 
+    def _spectral_density_model_lambda(wavelengths, **params):
```

Comment:
```suggestion
    def _spectral_density_model_wrapper(wavelengths, **params):
```
❔ :label: Since this isn't defined with a `lambda` statement, what would you think of renaming it as a wrapper?

Also I'm trying out prefixing code review comments with emojis!  I'm still figuring it out, but I'm using ❔ to indicate that it's optional and up to you, and :label: refers to naming.

Reply:
Oops, too late, looks like it's already merged but sure I'm fine with either name! I originally had it as a lambda, but switched to this syntax just to keep it under the line limit